sources/packages/web/src/components/ReadMoreContainer.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <slot></slot>
+    <div id="extendedContent" v-show="isExtended">
+      <slot name="extended"></slot>
+    </div>
+    <a id="toggleButton" @click.prevent="toggleExtended" href="#">{{
```

Comment:
is providing href=# mandatory?

Reply:
You are right about the "<a>" doesn't require the href attribute, but not use it has some implications for styling and more.
An "<a>" element without href is not accessible using the tab, for instance, hence I am keeping it.
https://stackoverflow.com/questions/10510191/valid-to-use-a-anchor-tag-without-href-attribute